### DISCUSSION TITLE
New suggestions and ideas for new releases

### DISCUSSION THREAD

**Discussion by Macronair** [Maerciez/TheMillionaireGame#58] [General]
Hi all,
I made this thread for you guys to submit some great suggestions and ideas that can be implemented in this project.
Lots of people already requested the following:

- Custom money tree (values & currency)
- More custom textures (also for questions)
- Video playback in TV screen.

If you have a new idea for the list (or maybe I forgot something), then let me know in this thread.

Thx!

**Comment by alplanche** [comment]
Hello ! I was looking for a Who Wants to Be a Millionaire software and yours is exactly what I needed ! It is super complete and, overall, fantastic !

In the font_sources folder, there is a font called ConduitITCBold that seems to be the one used in the show for the question and possible answers, but it seems not to be used by the software in that way.

So my suggestion is : To improve the fidelity of the graphics for the question and answers (use of ConduitITCBold font, add the little diamonds right before the letter of the possible answers, and set the colour of the diamond, letter and colon to orange).

**Comment by Smoozle** [comment]
Hey Macronair,

Absolutely legendary stuff you've got here. Looking to use this for a Charity event at some point soon.

Any chance of that custom money-tree being ready any time soon? <3

**Comment by ristoryti** [comment]
Hello!

Is there any chance of using the Fast Fingers feature on ios or android in the future? Or is there any workaround that you can think of?
It would greatly decrease the amount of computers needed for running a full game with many participants.

And thank you for making this project!

**Reply by Macronair** [reply]
Thanks for your suggestion ristoryti! This idea is also something that I want to make in order to play the Fastest Finger First part a lot easier with a lot less hardware. Now I'm currently looking for a solution to make a web-based FFF client that can work on multiple platforms. But at this time, my experience and knowledge with developing web apps (& mobile apps) is almost zero.

But I hope this will improve and there will be a better client in the future. ;)

**Comment by ImmanuelJeswyn** [comment]
Hi

Firstly, let me commend you on the excellent work you've done so far—I'm
truly impressed with the recent update enabling music customization. As
someone who frequently inquires about music-related features, I appreciate
your responsiveness and continuous improvement.

I have a suggestion that could further enhance the accessibility and appeal
of your software, particularly for users who intend to share their
creations on social media. If feasible, would you consider developing a
version with copyright-free music (CC0-licensed)? This would enable users
to use your software without concerns about copyright claims or strikes,
allowing them to seamlessly share their work across platforms.

Given the extensive music library in this software (which is good), finding
CC0 alternatives could be a challenging task. Perhaps we could create a
forum where users contribute and share links to suitable CC0 music,
streamlining the process. I'd be happy to participate by researching and
recommending CC0 tracks that align with your project's style.

I believe that by removing music copyright constraints, your software would
quickly gain popularity across broader user groups.

Thank you for considering this suggestion, and please let me know if I can
assist in any way.

Warm regards,
Jeswyn

On Thu, 30 Nov 2023 at 20:12, Marco Loenen ***@***.***> wrote:

> Hi all,
> I made this thread for you guys to submit some great suggestions and ideas
> that can be implemented in this project.
> Lots of people already requested the following:
>
>    - Custom money tree (values & currency)
>    - More custom textures (also for questions)
>    - Video playback in TV screen.
>
> If you have a new idea for the list (or maybe I forgot something), then
> let me know in this thread.
>
> Thx!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Macronair/TheMillionaireGame/discussions/58>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVV6I62GGOLMOOFASLXI3V3YHCLPDAVCNFSM6AAAAABABJX7HWVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZVHEYDSOBZGQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
In Christ,
Jeswyn.

**Reply by Macronair** [reply]
Hi Jeswyn, this is very interesting and also something that will be quite helpful for the content creators.
Maybe we can make a new thread of discussion on this repository to collect this music and sounds?

**Reply by ImmanuelJeswyn** [reply]
Sure. I am all for it.

**Comment by ristoryti** [comment]
I would also love to see a feature that would allow importing questions from a spreadsheet. I tried editing the tables in SSMS but it was quite a complicated task.

Cheers!

**Reply by Macronair** [reply]
This will be possible in the upcoming version (1.2)!
I will also make a short tutorial or guide for how to do this.

**Reply by ristoryti** [reply]
Amazing! Can't wait! Do you have an estimate, is the release of next version weeks or months away?

Thanks

**Comment by davemachala** [comment]
Hi, thank you so much for your work. I would love to be able to control the game using keyboard so I don't have to use a mouse. This would be useful for using StreamDeck or similar device. Thanks!

**Reply by Baumhausbanane123e** [reply]
That's what I was about to suggest too!

**Comment by Baumhausbanane123e** [comment]
Hi,
Would be cool if the main window of the programm could get a full screen too.
And it would be cool if sounds/music would fade out on stop instead of harshly stopping.
Thanks!

**Comment by Nolo337** [comment]
Dear Marco,

I downloaded and installed your game a month ago, and love using it with friends and family. It is so well-designed visually, and adds a whole new touch to the game. Well done, well structured! :)
May I have 2 ideas to improve the game and make it more customized?

Idea no. 1 - Question set/question stock
-Basically, some people get a bit sad when they get questions from only one topic. Do you think you could make a question set/stock opportunity available? The idea is: from the already created questions, I could make a "stock" or "set", a list of 15 questions handpicked by the owner to make sure to give questions mostly enjoyable per person. And also, the best would be if I could make endless amount of these "sets" saved to use them whenever they become handy. After a question from such a "set" or "stock" would be used, it is going to marked normally into "Question is used" status.
This would make the game for my friends not only tremendously customizable, but also more enjoyable, if I can load the set of questions according to their own interests, if possible.

Idea no. 2 - notes pop-up
I make extensive notes for selected questions. But I always need to open the question editor when such a question is played.
Do you think you would be able to make the notes appear as soon as a player picked a potential answer? That the information somehow pops-up on the main control panel?
I would greatly cherish this.

Please let me know if I should address ideas at another platform! Looking forward to hear from you!

Warm regards,
Daniel
